### DISCUSSION TITLE
docu website without blog stuff

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#71] [Q&A] [answered]
i use the start theme for different personal websites. My new project is a pure documetation site, i do not net the blog stuff. There is only a docs section.
I would like to start directly with the docs section and not with the normal start page with the latest posts....

Is there a way to do that?

**Comment by razonyang** [comment]
> I would like to start directly with the docs section and not with the normal start page with the latest posts....

If I understand correctly, you only want to customize the homepage? If so, you can achieve this via `landing` layout. To do so, create `content/_index.md` with the following content.

```markdown
---
title: "Hello world."
description: "Blah blah."
layout: landing
full_width: true # whether the current page take full width all the time.
---

THEN YOU CAN WRITE CONTENT HERE WITH SHORTCODES AND EVEN RAW HTML.
```

Please checkout the https://bootstrap.hugomods.com/docs/ for existing Bootstrap shortcodes, you can also create new shortcode to meet your needs.

| Example | Preview |
| --- | --- |
| https://raw.githubusercontent.com/hugomods/site/main/images/content/_index.md | https://images.hugomods.com/ |
| https://raw.githubusercontent.com/hugomods/site/main/docker/content/_index.md | https://docker.hugomods.com/ |

**Reply by Gordolin** [reply]
hi, thank you, landing page worked. Thank you for the examples, so it was a little bit easier for me to set up. 

I hoped, do that without a landing page and start directly with the docs site.
In default (my) setup the start page of the website ist the same as the posts site.
(because i have no public site, i can't show an example)

**Reply by razonyang** [reply, marked as answer]
> I hoped, do that without a landing page and start directly with the docs site.

There is no universal homepage for the docs site, since the docs's content is __diverse__, using landing layout with shortcodes is best way for now.

**Comment by razonyang** [comment]
Additional information

1. To remove archives, then delete the `content/archives` folder.
2. To disable sidebar, then remove all sidebar's modules which located in `hugo.yaml` and `params.yaml`.
3. To disable taxonomies (authors, tags, series, categories), then tweak the `taxonomies` in `hugo.yaml`.
4. ...
